assembly/__tests__/json.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { JSON } from '..';
+
+const jsonStr = '{"hello":"world"}';
+
+
+
+
+describe("JSON", (): void => {
+
```

Comment:
nit: no need for empty line as nesting is already pretty good separator from previous line